docs/installation.rst

```diff
@@ -9,14 +9,18 @@ you may want to use trusted certificate and other enhancement.
 Prerequisites
 =============
 
-In order to run temBoard agent, you require:
+In order to run temBoard agent, you need:
 
-- PostgreSQL 9.4+, listening on UNIX socket, on the same host. Check with ``sudo -u postgres psql``.
+- PostgreSQL 9.4+, running on Linux and listening on UNIX socket. Check with ``sudo -u postgres psql``.
 - openssl.
 - Python 2.6+ or 3.5+. Check with ``python --version``.
 - A running temBoard UI.
 - bash and sudo for setup script.
 
+.. note::
+
+  Any configuration other than the agent on the same Linux server than the PostgreSQL instance is not supported. The agent will refuse to run.
```

Comment:
Reformulation : "temBoard agent must run on same host as PostgreSQL instance. Running agent on remote host is not supported yet."

Reply:
Done in 2bd89a41c6daec31496b604bba5e2007485dadd5